bolt-design-system/bolt
# Fix: bolt-button text and icon alignment

## Jira

http://vjira2:8080/browse/BDS-791

## Summary

Improves the vertical alignment of text and icon inside `bolt-button`.

## Details

1. CSS are rewritten to handle vertical alignment of the text and icon. 
2. Icon-only option now has consistent spacing and size (perfect square/circle), and the icon inside will scale based on button size.
3. Documentation is rewritten to demonstrate each prop properly and clearly.
4. Folder structure and internal variables have been converted to current standards.
5. There should be no breaking changes, future tasks have been created in JIRA to cover further refactoring and cleanup that would break existing buttons.

## How to test

Run the branch locally and check to see if all the demos are working as intended.
